src/cnaas_nms/api/mgmtdomain.py

```diff
@@ -53,6 +55,23 @@ def ipv4_gw_valid_address(cls, v, values, **kwargs):
 
         return v
 
+    @validator('ipv6_gw')
+    def ipv6_gw_valid_address(cls, v, values, **kwargs):
+        try:
+            addr = IPv6Interface(v)
+            prefix_len = int(addr.network.prefixlen)
+        except:
+            raise ValueError('Invalid ipv6_gw received. Must be correct IPv6 address with mask')
+        else:
+            if addr.ip == addr.network.network_address:
+                raise ValueError("Specify gateway address, not subnet address")
+            if addr.ip == addr.network.broadcast_address:
+                raise ValueError("Specify gateway address, not broadcast address")
+            if prefix_len >= 126 or prefix_len <= 63:
```

Comment:
These values are my wild assumptions. Some feedback from @vidarstokke would be appreciated :)

Reply:
I agree. These are good values.